tests/api.py

```diff
@@ -52,6 +52,13 @@ def api_backend():
     del ctx
 
 
+def api_debug():
+    ctx = ngl.Context()
+    ret = ctx.configure(ngl.Config(offscreen=True, width=16, height=16, debug=True, backend=_backend))
```

Comment:
Shouldn't pass `debug=True` to all tests (`pynopegl-utils/pynopegl_utils/tests/cmp.py`) so that it fails in the CI?

Reply:
Building with gl debugging enabled, automatically enable the debug flag even if it is not requested by the user.